src/main/resources/kibana-rest.yml

```diff
@@ -22,6 +22,10 @@ kibana_alerts:
   versions:
     ">= 7.9.0": "/api/alerts/_find"
 
+kibana_alerts_health:
+  versions:
+    ">= 7.11.0": "/api/alerts/_health"
```

Comment:
Is this deprecated now (from 7.13) per [this doc](https://www.elastic.co/guide/en/kibana/current/alerts-api.html#alerts-api-health)?

Reply:
Bump @CamiloSierraH 🙏🏼